frontend/packages/dev-console/integration-tests/features/monitoring/Changing_context_in_Monitoring_Dashboard_3898

```diff
@@ -0,0 +1,29 @@
+Feature: Changing context in Monitoring Dashboard
+	User should be able I want to change the filter in the Monitoring Dashboard
+
+Background:
+    Given user logged into the openshift application
+    And user is at the developer perspective
+
+@regression
+Scenario: Checking filter dropdown for workloads on Monitoring Dashboard navigating from workloads
+   Given user is in topology view with workloads
+   When user clicks monitoring dashboard from the sidebar of workload
+   And user checks filter dropdown by default have "All Workloads" selected
+   And user clicks on dropdown
+   And user selects one workload
+   And user checks for charts of the workload
+   And user refreshes the page
+   Then the page will auto select the previously selected workload
+
+@regression
+Scenario: Checking filter dropdown for workloads on Monitoring Dashboard navigating from nav bar
+   Given User is at the Monitoring dashboard
+   When user clicks on filter dropdown
+   And user selects "All Workloads"
+   And user can check charts for all workloads
```

Comment:
This is something an outcome I believe.. Can you split the scenarios as required